vllm/distributed/device_communicators/cuda_wrapper.py

```diff
@@ -60,6 +62,29 @@ def find_loaded_library(lib_name) -> Optional[str]:
     return path
 
 
+def get_cudart_lib_path_from_env() -> Optional[str]:
+    """
+    In some system, find_loaded_library() may not work. So we allow users to
+    specify the path through environment variable VLLM_CUDART_SO_PATH.
+    """
+    cudart_so_env = envs.VLLM_CUDART_SO_PATH
+    if cudart_so_env is not None:
+        cudart_paths = [
+            cudart_so_env,
+        ]
+        for path in cudart_paths:
+            file_paths = glob.glob(path)
```

Comment:
why is it a `glob` ? I think it should just be a path that can be used?

Reply:
Good catch, cleaned up in https://github.com/vllm-project/vllm/pull/13203.

We were thinking specify some partial path. But using the single path is cleaner. Thanks for pointing it out.